.circleci/config.yml

```
@@ -65,13 +65,13 @@ references:
   restore-tsc-cache: &restore-tsc-cache
     name: Restore TypeScript cache
     keys:
-      - v{{ .Environment.GLOBAL_CACHE_PREFIX }}-v0-tsc-{{ .Branch }}-{{ .Revision }}
-      - v{{ .Environment.GLOBAL_CACHE_PREFIX }}-v0-tsc-{{ .Branch }}
-      - v{{ .Environment.GLOBAL_CACHE_PREFIX }}-v0-tsc-master
-      - v{{ .Environment.GLOBAL_CACHE_PREFIX }}-v0-tsc
+      - v{{ .Environment.GLOBAL_CACHE_PREFIX }}-v1-tsc-{{ .Branch }}-{{ .Revision }}
+      - v{{ .Environment.GLOBAL_CACHE_PREFIX }}-v1-tsc-{{ .Branch }}
+      - v{{ .Environment.GLOBAL_CACHE_PREFIX }}-v1-tsc-master
+      - v{{ .Environment.GLOBAL_CACHE_PREFIX }}-v1-tsc
   save-tsc-cache: &save-tsc-cache
     name: Save TypeScript cache
-    key: v{{ .Environment.GLOBAL_CACHE_PREFIX }}-v0-tsc-{{ .Branch }}-{{ .Revision }}
+    key: v{{ .Environment.GLOBAL_CACHE_PREFIX }}-v1-tsc-{{ .Branch }}-{{ .Revision }}
     paths:
       - ~/.tsc-cache
```

Interesting one here… for composite and incremental builds, I believe TypeScript skips outputting anything if the caches are up to date. We can add `packages/*/dist` to the cache to catch and `tsc`-build output.
```suggestion
      - ~/.tsc-cache
      - packages/*/dist
```